Source/Core/DolphinQt2/Config/Graphics/HacksWidget.cpp

```diff
@@ -98,6 +101,12 @@ void HacksWidget::CreateWidgets()
   setLayout(main_layout);
 }
 
+void HacksWidget::OnBackendChanged()
+{
+  m_gpu_texture_decoding->setEnabled(g_Config.backend_info.bSupportsGPUTextureDecoding);
```

Comment:
These should also be done when the widgets are created.

Reply:
It is now.